tools/data_converter/s3dis_data_utils.py

```diff
@@ -99,15 +107,51 @@ def process_single_scene(sample_idx):
                 'instance_mask', f'{self.split}_{sample_idx}.bin')
             info['pts_semantic_mask_path'] = osp.join(
                 'semantic_mask', f'{self.split}_{sample_idx}.bin')
+            info['annos'] = self.get_bboxes(points, pts_instance_mask,
+                                            pts_semantic_mask)
 
             return info
 
         sample_id_list = sample_id_list if sample_id_list is not None \
             else self.sample_id_list
-        with futures.ThreadPoolExecutor(num_workers) as executor:
+        with futures.ThreadPoolExecutor(1) as executor:
             infos = executor.map(process_single_scene, sample_id_list)
         return list(infos)
 
+    def get_bboxes(self, points, pts_instance_mask, pts_semantic_mask):
+        """Convert instance masks to axis-aligned bounding boxes.
+
+        Args:
+            points (np.array): Scene points of shape (n, 6).
+            pts_instance_mask (np.array): Instance labels of shape (n,).
+            pts_semantic_mask (np.array): Semantic labels of shape (n,).
+
+        Returns:
+            infos (dict): Bounding boxes with labels.
```

Comment:
For the returned values, we often don't write its name here. I think the following docstring is better?
```
Returns:
    dict: A dict containing detection infos with following keys:

        - gt_boxes_upright_depth (np.ndarray): Bounding boxes of shape (n, 6)
        - class (np.ndarray): Box labels of shape (n,)
        - gt_num (int): Number of boxes.
```